### ISSUE TITLE
NullReferenceException in GetModels()

### ISSUE THREAD

**Issue by cguedel** [Clarifai/clarifai-csharp#6]
We're trying to get the models available in a system by using:

```
var client = new ClarifaiClient(configuration.ApiToken);
var models = await client.GetModels().ExecuteAsync().ConfigureAwait(false);
```

however, this throws 

```
System.NullReferenceException: Object reference not set to an instance of an object.
   at Clarifai.API.Requests.Models.GetModelsRequest.Unmarshaller(Object jsonObject)
   at Clarifai.API.Requests.ClarifaiRequest`1.ExecuteAsync()
   ...
```

The same also happens when running the integration test for this method, which then points to `ModelType.DetermineModelType((string)model.output_info.type_ext)` returning `null` for `{"type_ext": "ocr-detect"}`

This makes this effectively unusable for us.

**Comment by rok-povsic** [comment]
Hello @cguedel we've released a version 1.3.2 where this is fixed.

Thank you for reporting the issue.

**Comment by cguedel** [comment]
Thanks for the fix @rok-povsic 
Our services now log `Warning: Unknown model type 'ocr-detect', skipping. Please upgrade the to the latest version of the library.` on the console, just as a FYI
